addon/styles/components/notification-message.css

```diff
@@ -48,11 +48,11 @@
   justify-content: space-between;
   padding: --spacing-1 --spacing-2;
   word-break: break-word;
+}
 
-  & a {
-    color: #fff;
-    text-decoration: underline;
-  }
+a.c-notification__content {
```

Comment:
Oops, looks like this is actually wrong. Should've been `.c-notification__content a`